docs/localization/LOCALIZATION.md

```diff
@@ -74,343 +80,430 @@ export function localizationConfigurationFactory() {
 
-You can specify a **language** in the configuration, it will be loaded at the bootstrap time of you application.
-A language can be specified asynchronously by using the **LocalizationService.useLanguage** function.
+You can specify a **language** in the configuration, it will be loaded at the bootstrap time of your application.
+A language can be specified asynchronously by using the `LocalizationService.useLanguage(...)` function.
 
-- In case it is not enough to provide your configuration object as explained above, you can also provide a new factory for the localization configuration. You only need to provide a token `LOCALIZATION_CONFIGURATION_TOKEN` with you custom factory and dependencies.
+- In case it is not enough to provide your configuration object as explained above, you can also provide a new factory for the localization configuration.
+  You only need to provide a token `LOCALIZATION_CONFIGURATION_TOKEN` with your custom factory and dependencies.
 
 ```typescript
-// index.ts
-
-import { LOCALIZATION_CONFIGURATION_TOKEN, LocalizationModule, translateLoaderProvider } from  "@o3r/localization";
+// app.module.ts
 
-import {BidiModule} from '@angular/cdk/bidi';
-import { TranslateModule } from  "@ngx-translate/core";
-...
-import localeAR from  "@angular/common/locales/ar";
-import localeEN from  "@angular/common/locales/en";
-import localeFR from  "@angular/common/locales/fr";
-
-
-...
+import localeAR from '@angular/common/locales/ar';
+import localeEN from '@angular/common/locales/en';
+import localeFR from '@angular/common/locales/fr';
+import { TranslateModule } from '@ngx-translate/core';
+import { LOCALIZATION_CONFIGURATION_TOKEN, LocalizationModule, translateLoaderProvider } from '@o3r/localization';
+// ...
 
 registerLocaleData(localeEN, 'en-EN');
 registerLocaleData(localeFR, 'fr');
 registerLocaleData(localeAR, 'ar');
 
 @NgModule({
   imports: [
-  LocalizationModule,
-  BidiModule,
-  TranslateModule.forRoot({
-    loader: translateLoaderProvider,
-  }),
-  ...
+    LocalizationModule,
+    TranslateModule.forRoot({
+      loader: translateLoaderProvider,
+    })
+    // ...
   ],
-  ...
+  // ...
   providers: [
     {provide: LOCALIZATION_CONFIGURATION_TOKEN, useFactory: customCreateLocalizationConfiguration, deps: [YourServiceNeededByTheFactory]}
   ]
+})
+class AppModule {}
 ```
 
-You may also need to set up your http server to accept **CORS** and update **connect-src** property of **csp.json** in your application.
+## HTTP server
 
-```typescript
+You may also need to set up your http server to accept **CORS** and update the `connect-src` property of `csp.json` in your application.
 
+```json5
 // configs/csp.json
 
 {
+  "connect-src": "http://example.com"
+}
+```
 
-"connect-src": "http://example.com"
+## Localizing the components
 
-}
+Now we are ready to localize the components of the application.
+
+### How to generate localization files for a component
```

Comment:
I think this session is missing information regarding 2 points:
- The `--use-localization` is mandatory only if the parameter is not provided in the workspace config file (angular.json, nx.json or project.json) under the schematics parameters property.
- The `ngAdd` of the localition package will set this parameter

(Not sure there is documentation regarding this parameter neither :S)